meta-dts-distro/recipes-bsp/txe-secure-boot/smmstoretool_git.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "Offline SMMSTORE variable modification too"
+LICENSE = "GPL-2.0-or-later"
+
+SRC_URI = " \
+git://github.com/coreboot/coreboot.git;branch=main;protocol=https \
```

Comment:
isn't main the default branch anyway?

Reply:
I think BitBake searches `master` by default. `coreboot` has `main` instead, so the fetcher fails if the branch is not provided
```
ERROR: smmstoretool-git-r0 do_fetch: Fetcher failure: Unable to find revision 602653abed391ae1b1445ad86d0f05b8b5b678cb in branch master even from upstream
```